### DISCUSSION TITLE
Is this still working?

### DISCUSSION THREAD

**Discussion by asgeirtj** [xiaoxiaoflood/firefox-scripts#318] [General Support]
Landed here on my quest for extension manager on Firefox to be able to quickly toggle on or off addons. 

![image](https://github.com/user-attachments/assets/0bb2cd7d-f3ab-4e9a-a63b-867539abb324)
![image](https://github.com/user-attachments/assets/7a6d5c37-5de7-44ec-a4fd-7b42a85592f8)
![image](https://github.com/user-attachments/assets/ccaab731-c246-4427-ad87-89c6f74aaa10)
![image](https://github.com/user-attachments/assets/ff638838-5994-4ad8-9573-68a87ca0d49e)
![image](https://github.com/user-attachments/assets/53af704f-c3b7-418b-b41e-e2344853ca56)
![image](https://github.com/user-attachments/assets/53b4b2d4-ae1b-4ca1-97c1-217fc716a60e)

Was expecting to see some kind of extensionOptionsMenu.uc.js plugin but don't see it. Tried with Firefox newest verison and Firefox Dev 131.0b8 (aarch64 and 130.0.1 (64-bit)

**Comment by xiaoxiaoflood** [comment]
> Is this still working?

https://github.com/xiaoxiaoflood/firefox-scripts/blob/ac32bd96101c91e0926e5068ff4aeef543d056c3/README.md?plain=1#L3

Scripts (.uc.js files) must be placed in chrome folder, not in utils subfolder. utils is for files such as userChrome.jsm and chrome.manifest.

> Save the desired [userChromeJS scripts](https://github.com/xiaoxiaoflood/firefox-scripts/tree/master/chrome) into **chrome**.

I must say I don't test on macOS, so I hope the directories still work there.

**Comment by xiaoxiaoflood** [comment]
And make sure to always have the latest version for every file. I see from your screenshots, for instance, your userChrome.jsm is outdated, it's from 2021, but it was updated past month.

**Reply by asgeirtj** [reply]
thx so much
